android-base/src/main/java/io/github/droidkaigi/confsched2020/initializer/AppInitializers.kt

```diff
@@ -0,0 +1,14 @@
+package io.github.droidkaigi.confsched2020.initializer
+
+import android.app.Application
+import javax.inject.Inject
+
+class AppInitializers @Inject constructor(
+    private val initializers: Set<@JvmSuppressWildcards AppInitializer>
```

Comment:
Is `@JvmSuppressWildcards` still needed? 👀 
https://youtu.be/o-ins1nvbDg?t=655

Reply:
@takahirom I understand that the above feature is not supported.
```
e: /.../android-base/build/tmp/kapt3/stubs/debug/io/github/droidkaigi/confsched2020/di/AppComponent.java:8: error: [Dagger/MissingBinding] java.util.Set<? extends io.github.droidkaigi.confsched2020.initializer.AppInitializer> cannot be provided without an @Provides-annotated method.
public abstract interface AppComponent extends dagger.android.AndroidInjector<io.github.droidkaigi.confsched2020.App>, io.github.droidkaigi.confsched2020.di.AppComponentInterface {
                ^
      java.util.Set<? extends io.github.droidkaigi.confsched2020.initializer.AppInitializer> is injected at
          io.github.droidkaigi.confsched2020.initializer.AppInitializers(initializers)
      io.github.droidkaigi.confsched2020.initializer.AppInitializers is injected at
          io.github.droidkaigi.confsched2020.App.initializers
      io.github.droidkaigi.confsched2020.App is injected at
          io.github.droidkaigi.confsched2020.di.AppComponent.inject(io.github.droidkaigi.confsched2020.App)
```